doc/ref_impl/cntt-ri/chapters/chapter07.md

```diff
@@ -5,15 +5,84 @@
 
 ## Table of Contents
 * [7.1 Introduction](#7.1)
-* [7.2 Heading](#7.2)
-* [7.3 Heading](#7.3)
-
+* [7.2 Pre-requisites](#7.2)
+* [7.3 Requirements Gathering](#7.3)
+* [7.4 Access and Connectivity](#7.4)
+* [7.5 Descriptor File Preparations](#7.5)
+* [7.6 Deployment Installer & Install Steps](#7.6)
+* [7.7 Deployment Validations](#7.7)
+* [7.8 CICD Tool Chain (use of, process, and scripts)](#7.8)
+* [7.9 Jenkins Setup & Job Creation](#7.9)
+* [7.10 Compliance Validation (steps, process)](#7.10)
+ 
 <a name="7.1"></a>
 ## 7.1 Introduction
 
+The purpose of this chapter is to establish an operational run-book containing sequences and steps explaining, with enough detail, how to install a Reference Implementation (RI).
+
+The term Run-Book is synonymous with "Cookbook", and either term will be used interchangeably to indicate that if a user follows the steps or procedures in the "book", the expected result will be an RI identical to the lab utilized by CNTT for the instation and certificaiton of RI-1.
```

Comment:
"... the expected result will be an RI identical to the lab utilized by CNTT for the instation and certificaiton of RI-1" -- will the run-book not be utilised in the installation of RI-1?

Typos: instantiation and certification.